tutorials/snap/using-intel-realsense-sdk.md

```diff
@@ -0,0 +1,328 @@
+
+  - **Tools**:
+     - **Capture tool**: Provides a GUI to view camera streams, create a new file from a live camera, and play a file in the supported formats. The tool provides options to render the camera or file images.
+     - **Projection tool**: Provides simple visualization of the projection functions output, to allow human eye detection of major offsets in the projection computation.
+     - **System Info tool**: Presents system data such as Linux name, Linux kernel version, CPU information, and more.
+
+  - **Utilities**:
+     - **Log**: The SDK provides a logging library, which can be used by the SDK components and the application to log meaningful events. 
+     - **Time Sync utility**: Provides methods to synchronize multiple streams of images and motion samples based on the samples time-stamp or sample number. 
+     - **SDK Data Path utility**: The SDK provides a utility to locate SDK files in the system.
+     The utility is used by Computer Vision modules, which need to locate data files in the system that are constant for all applications (not application- or algorithm-instance specific).
+     - **FPS counter**:  Measures the actual FPS in a specific period. You can use this utility to check the actual FPS in all software stack layers in your applications, to analyze FPS latency in those layers.
+
+
+#### realsense_samples-v0.6.2.tar.gz
+
+We are going to detail the available samples in the next section!
+
+## Available samples
+Duration: 30:00
+
+### Preliminary notes
+
+On ubuntu desktop, you can run any examples you want (graphical and non graphical).  Most of the samples are rendering some graphical UI. Those have the **rs_** prefix for realsense SDK and **cpp_** prefix for those using librealsense (the bare protocole).
+
+Ubuntu Core, on itself, doesn’t have any graphical server by default. Consequently, you can’t run most of the samples rendering some graphical UI. However, there are a lot of samples with a **_web** appendix. They are spawning a web interface that you can access to the network, showing you real time data and capture from the camera.
+
+You can focus and experiment on both kind of examples.
+
+### Running your first sample
+Here is an example with **rs_pt_tutorial_1_web**:
+
+
+```bash
+$ rs_pt_tutorial_1_web 
+
+enabling: stream:DEPTH, 320x240x30, format:Z16
+
+enabling: stream:COLOR, 320x240x30, format:RGB8
+Device open failed, aborting...
+Device open failed, aborting...
+cl_get_gt_device(): error, unknown device: ffffffff
+
+web folder path: /usr/share/librealsense/samples/rs_pt_tutorial_1_web_browser
+
+Server: waiting for client to connect...
+
+Best host: 192.168.0.29 of ifa type: wlan0
+ >>> point your browser to: http://192.168.0.29:8000/view.html
+```
+
+Then, point your browser to [http://localhost:8000/view.html] (or a remote IP if you are trying to browser from a different machine) and you will get something like:
```

Comment:
the text in brackets isn't rendering because the markdown parser thinks this is a link.